tasks/start-performance-infrastructure.yml

```diff
@@ -0,0 +1,30 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: eu.gcr.io/census-gcr/gcloud-kubectl
+params:
+  ENV:
+  SERVICE_ACCOUNT_JSON:
+  KUBERNETES_CLUSTER:
+  GCP_PROJECT_NAME:
+inputs:
+  - name: census-rm-kubernetes-dependencies-repo
+    path: bash
+    args:
+      - -exc
+      - |
+        cat >~/gcloud-service-key.json <<EOL
+        $SERVICE_ACCOUNT_JSON
+        EOL
+        # Use gcloud service account to configure kubectl
+        gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+        gcloud container clusters get-credentials ${KUBERNETES_CLUSTER} --zone europe-west2 --project ${GCP_PROJECT_NAME}
+        
+        # Start Postgresql instance
+        SQL_NAME=`gcloud sql instances list --format="value(name)"`
+        echo $SQL_NAME
+        gcloud sql instances patch $SQL_NAME --activation-policy ALWAYS
+        # Scale up cluster
+        yes | gcloud container clusters resize $(KUBERNETES_CLUSTER) --num-nodes 2 --node-pool=default-node-pool --region=europe-west2
```

Comment:
The node number doesn't match with what's currently in our performance environment. Should this match?

Reply:
Forgot to ask about this in standup yesterday (🤦‍♂) so will ask today